registry/application/access.py

```diff
@@ -1,29 +1,14 @@
-from enum import Enum
-
+from registry.constants import Role, Action
 from registry.infrastructure.repositories.organization_repository import OrganizationRepository
 
-
-class Role(Enum):
-    OWNER = "OWNER"
-    MEMBER = "MEMBER"
-
```

Comment:
@keshrisohit moved it to constant as importing `Role` was causing import loop in files